src/index.ts

```diff
@@ -77,4 +85,35 @@ export default class BlueButton {
           : SANDBOX_BASE_URL,
     };
   }
+
+  public getAccessTokenUrl(): string {
+    return this.baseUrl + "/" + this.version + "/o/token/";
```

Comment:
```suggestion
    return `${this.baseUrl}/${this.version}/o/token/`;
```
Just for consistency sake

Reply:
Updated this and verified other areas are changed over to this method. TY!